x/delegation/keeper/logic_delegation.go

```diff
@@ -55,11 +59,14 @@ func (k Keeper) performUndelegation(ctx sdk.Context, stakerAddress string, deleg
 
 // performWithdrawal withdraws all pending rewards from a user and transfers it.
 // The amount is returned by the function.
-func (k Keeper) performWithdrawal(ctx sdk.Context, stakerAddress, delegatorAddress string) uint64 {
+func (k Keeper) performWithdrawal(ctx sdk.Context, stakerAddress, delegatorAddress string) (sdk.Coins, error) {
 	reward := k.f1WithdrawRewards(ctx, stakerAddress, delegatorAddress)
-	err := util.TransferFromModuleToAddress(k.bankKeeper, ctx, types.ModuleName, delegatorAddress, reward)
-	if err != nil {
-		util.PanicHalt(k.upgradeKeeper, ctx, "no money left in module")
+	recipient, errAddress := sdk.AccAddressFromBech32(delegatorAddress)
```

Comment:
what about `sdk.MustAccAddressFromBech32`?

Reply:
It could panic and I dont now if stakerAddress is always sanitized.